### DISCUSSION TITLE
Supply Chain Tracing and Blockchain

### DISCUSSION THREAD

**Discussion by rufuspollock** [rufuspollock/awesome-crypto-critique#75] [General]
Trying to understand the #crypto / #web3 story about supply chain tracing and auditing.

Am I right that the key issue is a "double spend" problem?

And how is #blockchain #DLT relevant and/or better than other solutions e.g. auditing, self-publishing #opendata?

![image](https://user-images.githubusercontent.com/180658/158958194-3b3bddc6-51b9-45c7-aeb9-37ec0e3b4e9c.png)

Coggle (20% done): https://coggle.it/diagram/YjNTrT6jBM9du7n3/t/supply-chain-tracing-re-blockchain

**Comment by edent** [comment]
I think there are a few unsolvable problems.

The first is - what's the user story? Let's assume it is as straightforward as "As a buyer of product X, I want to know that it is genuine."

That's relatively simple on a "single" product. I scan the QR on my designer handbag and it says that it is both genuine and that no-one else has claimed this item.  But even that is somewhat difficult.  A real code can be copied and placed onto a fake bag. So the seller can keep the genuine item and sell the knock-off. Sure, the seller can't sell the same thing multiple times - but they get to keep the real thing.

Given that there are multiple blockchains, how does the buyer know to trust the destination of the code? Will they be able to tell that it's `GUCCI` vs `GUCCl`?

But let's say the user scans the code and claims ownership of the product. What happens if the user loses access to their wallet - does the item become unsellable? What if the QR falls off or gets damaged? 

With commodities, it becomes even more difficult. Let's say you have a pallet of goods. Each person in the chain of custody asserts on a blockchain that they have taken control of it. One of them cracks open the crate and replaces some of the goods with fakes. How can you tell which one?

If every single transaction ever is recorded, then you *might* be able to see that supplier Y suddenly had more of the commodity than they should. But that assumes no one is ever going to do a cash-in-hand off-the-books transaction.

Finally, could this *only* be solved by a blockchain? If so, why would everyone agreeing to use a single chain be better than everyone using a single centralised supplier?

Complaints about dodgy commodities have a long - very long - history. See https://en.wikipedia.org/wiki/Complaint_tablet_to_Ea-nasir

**Reply by rufuspollock** [reply]
@edent excellent set of points.
